libs/agent-sm/bootstrap/src/main/java/org/opensearch/javaagent/bootstrap/AgentPolicy.java

```diff
@@ -37,20 +122,24 @@ private AgentPolicy() {}
      * @param policy policy
      */
     public static void setPolicy(Policy policy) {
-        setPolicy(policy, Set.of(), new String[0]);
+        setPolicy(policy, Set.of(), new AllCanExit());
```

Comment:
Why default this to AllCanExit?

Reply:
This is a good question, I think the default should be `NoneCanExit`, changing that, thank you